site/content/installation/ngf-images/jwt-token-docker-secret.md

```diff
@@ -0,0 +1,171 @@
+
+{{< /note >}}
+
+## Before You Begin
+
+You will need the following information from [MyF5](https://my.f5.com) for these steps:
+
+- A JWT Access Token (per instance) for NGINX Gateway Fabric from an active NGINX Gateway Fabric subscription.
+- The certificate (`nginx-repo.crt`) and key (`nginx-repo.key`) for each NGINX Gateway Fabric instance, used to list the available image tags from the Docker registry API.
+
+## Prepare NGINX Gateway Fabric
+
+1. Choose your desired [NGINX Gateway Fabric Image](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/CHANGELOG.md).
+1. Log into the [MyF5 Portal](https://my.f5.com/), navigate to your subscription details, and download the relevant .cert, .key and .JWT files.
+1. Create a Kubernetes secret using the JWT token. You should use `cat` to view the contents of the JWT token and store the output for use in later steps.
+1. Ensure there are no additional characters or extra whitespace that might have been accidentally added. This will break authorization and prevent the NGINX Gateway Fabric image from being downloaded.
+1. Modify your deployment (manifest or Helm) to use the Kubernetes secret created in step 3.
+1. Deploy NGINX Gateway Fabric into your Kubernetes cluster and verify that the installation has been successful.
+
+## Using the JWT token in a Docker Config Secret
+
+1. Create a Kubernetes `docker-registry` secret type on the cluster, using the JWT token as the username and `none` for password (as the password is not used).  The name of the docker server is `private-registry.nginx.com`.
+
+    ```shell
+    kubectl create secret docker-registry regcred --docker-server=private-registry.nginx.com --docker-username=<JWT Token> --docker-password=none [-n nginx-gateway]
+    ```
+
+    It is important that the `--docker-username=<JWT Token>` contains the contents of the token and is not pointing to the token itself. Ensure that when you copy the contents of the JWT token, there are no additional characters or extra whitespaces. This can invalidate the token and cause 401 errors when trying to authenticate to the registry.
+
+1. Confirm the details of the created secret by running:
+
+    ```shell
+    kubectl get secret regcred --output=yaml
+    ```
+
+1. You can now use the newly created Kubernetes secret in Helm and manifest deployments.
+
+---
+
+## Manifest Deployment
+
+The page ["Installation with Manifests"](https://docs.nginx.com/nginx-gateway-fabric/installation/installing-ngf/manifests/) explains how to install NGINX Gateway Fabric using manifests. The following snippet is an example of a deployment:
+
+```yaml
+apiVersion: gateway.nginx.org/v1alpha1
+kind: NginxGateway
+metadata:
+  name: nginx-gateway-config
+  namespace: nginx-gateway
+  labels:
+    app.kubernetes.io/name: nginx-gateway
+    app.kubernetes.io/instance: nginx-gateway
+    app.kubernetes.io/version: "edge"
+spec:
+  logging:
+    level: info
+```
```

Comment:
Not sure why this yaml is here.

Reply:
this is an example of generated manifests. NIC docs were similarly structured so I added it

Reply:
have removed these manifests already